antares-version.json

```diff
@@ -0,0 +1,3 @@
+{
+    "antares_deps_version": "2.0.0-rc2"
+}
```

Comment:
I'm not sure why we need to keep track of antares_deps_version in a .json file.

If it's a submodule, the branch name is kept in file **.gitmodules**.

```ini
[submodule "src/antares-deps"]
	path = src/antares-deps
	url = https://github.com/AntaresSimulatorTeam/antares-deps.git
	branch = v1.0.2
```

Reply:
It's used in antares-xpansion to avoid definition of version in each github action .yml files.
We could look for github-action to get submodules tag.
⚠️ In this case we should only point to release tag because otherwise there won't be any release asset available to download